src/bugle_forms/handlers/user.clj

```diff
@@ -8,17 +8,33 @@
 
 (defn login
   "Display login form."
-  [{:keys [flash]}]
-  (response/response (layout/application
-                      {:title "Log In" :flash flash}
-                      "Stub for log-in")))
+  [{:keys [flash], {:keys [user]} :session}]
+  (if user
+    (response/redirect "/dashboard" :see-other)
+    (response/response (layout/application
+                        {:title "Log In" :flash flash :user user}
+                        user-views/login))))
 
 (defn signup
   "Display signup form."
-  [{:keys [flash]}]
-  (response/response (layout/application
-                      {:title "Sign up" :flash flash}
-                      user-views/signup)))
+  [{:keys [flash], {:keys [user]} :session}]
+  (if user
+    (response/redirect "/dashboard" :see-other)
+    (response/response (layout/application
+                        {:title "Sign up" :flash flash}
+                        user-views/signup))))
+
+(defn create-session
```

Comment:
The name `create-session` is a bit confusing. It seems to suggest that the fn mostly takes care of creating session based on the current implementation, but here it's actually the handler for the POST request for login, which might or might not create the session based on credential verification.

Maybe rename this to `login` and the other GET handler to `login-form`?